doc/plutus/tutorials/basic-minting-policies.rst

```diff
@@ -1,51 +0,0 @@
-.. highlight:: haskell
-.. _basic_minting_policies_tutorial:
-
-Writing basic minting policies
-==============================
-
-:term:`Minting policy scripts<minting policy script>` are the programs that can be used to control the minting of new assets on the chain.
-Minting policy scripts are much like :term:`validator scripts<validator script>`, and they are written similarly, so check out the :ref:`basic validators tutorial<basic_validators_tutorial>` before reading this one .
-
-Minting policy arguments
-------------------------
-
-Minting policies, like validators, receive some information from the validating node:
-
-- The :term:`redeemer`, which is some script-specific data specified by the party performing the minting.
-- The :term:`minting context`, which contains a representation of the spending transaction, as well as the hash of the minting policy which is currently being run.
-
-The minting policy is a function which receives these two inputs as *arguments*.
-The validating node is responsible for passing them in and running the minting policy.
-As with validator scripts, the arguments are passed encoded as :hsobj:`PlutusCore.Data.Data`.
-
-Using the minting context
--------------------------
-
-..
-    TODO: pin down the naming: minting vs policy context
-
-Validators have access to the :term:`minting context` as their second argument.
-This will always be a value of type ``Ledger.Validation.PolicyCtx`` encoded as ``Data``.
-
-The minting context is very similar to the :term:`validation context`, and allows access to all the same features of the transaction.
-Minting policies tend to be particularly interested in the ``mint`` field, since the point of a minting policy is to control which tokens are minted.
-
-It is also important for a minting policy to look at the tokens in the ``mint`` field that are part of its own asset group.
-This requires the policy to refer to its own hash -- fortunately this is provided for us in the minting context.
-
-Here is an example that puts this together to make a simple policy that allows anyone to mint the token so long as they do it one token at a time.
-
-.. literalinclude:: BasicPolicies.hs
```

Comment:
`BasicPolicies.hs` should be removed too.

Reply:
Removed

Reply:
I've removed the file `basic-minting-policies.rst`, but just to make sure I don't get any build errors, I'm leaving the file `BasicPolicies.hs` because I don't think it causes any problems by remaining.

Reply:
I'd advice removing `BasicPolicies` if it's not needed. You can just run `cabal build plutus-doc` locally to test that it's not used anywhere.